demisto_sdk/commands/init/initiator.py

```diff
@@ -415,10 +415,14 @@ def find_secrets(self):
 
     def ignore_secrets(self, secrets):
         pack_dir = get_pack_name(self.full_output_path)
-        with open(f'Packs/{pack_dir}/.secrets-ignore', 'a') as f:
-            for secret in secrets:
-                f.write(secret)
-                f.write('\n')
+        try:
+            with open(f'Packs/{pack_dir}/.secrets-ignore', 'a') as f:
+                for secret in secrets:
+                    f.write(secret)
+                    f.write('\n')
+        except FileNotFoundError:
+            print_warning("Could not find the .secrets-ignore file - make sure your path is correct and is of format "
+                          "Packs/my_pack_name")
```

Comment:
can the user really make sure? isn't this an automatic thing that happens in the init process?

Reply:
Your'e right it's redundant. will remove